### ISSUE TITLE
GitHub repository does not link to the project website url

### ISSUE THREAD

**Issue by thisisobate** [kubeflow/community#692]
As part of our ongoing effort to [ensure all the websites within the CNCF meet the CLOMonitor guidelines](https://github.com/cncf/techdocs/issues/196), we noticed that Kubeflow does not pass the website criteria on [CLOMonitor](https://clomonitor.io/search?not_passing_check=website&foundation=cncf&page=1). 

To fix this:
Edit [kubeflow/community](https://github.com/kubeflow/community) repository details. See Notary for example: 
<img width="1704" alt="Screenshot 2023-11-22 at 09 29 19" src="https://github.com/notaryproject/notary/assets/29557702/0b9dd1e3-f32b-42c6-adfc-a860ebad347b">

In the website section, add the link to the Kubeflow website (https://www.kubeflow.org). See Notary for example:
<img width="1704" alt="Screenshot 2023-11-22 at 09 30 15" src="https://github.com/notaryproject/notary/assets/29557702/c883c1c3-f9d7-4fcc-ae3c-fadf402d43cd">

Once done, feel free to close this issue as CLOMonitor will recrawl the page and update accordingly.

**Comment by StefanoFioravanzo** [comment]
@thisisobate FYI our main "entrypoint" repo is https://github.com/kubeflow/kubeflow. I do see that most CNCF projects you link to point to \<project\>/community.

I see that even kubeflow/kubeflow doesn't link to the website. This is probably something that some from the steering commitee can quickly fix. cc @andreyvelich @james-jwu @jbottum @johnugeorge @terrytangyuan

**Comment by thisisobate** [comment]
Yeah...I understand `kubeflow/kubeflow` is the main entry point but for the clomonitor checks to pass, the update needs to be made in `kubeflow/community`.

It would be nice also for the `kubeflow/kubeflow` repo to be linked to the website.
